services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -38,15 +37,21 @@ class LinkPropertiesSpec extends BaseSpecification {
         northbound.getAllLinkProps().empty
     }
 
-    @Ignore("This test is not valid in this implementation.")
-    def "Valid error response is returned when sending invalid char in link props key"() {
+    def "Link props are created with empty values when sending not a valid link props key"() {
         when: "Send link property request with invalid character"
         def response = northbound.updateLinkProps([new LinkPropsDto("00:00:00:00:00:00:00:01", 1,
-                "00:00:00:00:00:00:00:02", 1, ['`cost': "700"])])
+                "00:00:00:00:00:00:00:02", 1, ["`cost": "700"])])
 
-        then: "Response states that operation failed"
-        response.failures == 1
-        response.messages.first() == "Invalid request"
+        then: "Response states that operation succeeded"
```

Comment:
I find this to be a defect and not an expected behavior. This was introduced during pre-H&S release and was considered a minor issue (low priority)

Reply:
Yes, I understand but I would like to verify that Kilda is not hanging as it was before. I will add a TODO with mention that it is a defect and in the future the test should be aligned with the right behavior.